FlightMatcher/FlightMatcher/Stories/FlightMatch/View/FlightMatchCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  FlightMatchCell.swift
+//  FlightMatcher
+//
+//  Created by Daria Korneichuk on 4/3/17.
+//  Copyright © 2017 Dmitry Smolyakov. All rights reserved.
+//
+
+import UIKit
+
+class FlightMatchCell: UITableViewCell {
+    
+    @IBOutlet weak var fromToLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var flightLabel: UILabel!
+    
+    class func reuseIdentifier() -> String {
+        return String(describing: FlightMatchCell.self)
+    }
+    
+    func configure(withItem item: Any) {
```

Comment:
Please try to avoid using `Any`. This is strong typed language and its not preferred to pass something around as `Any`.